iron-overlay-manager.html

```diff
@@ -0,0 +1,107 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+
+<script>
+
+  Polymer.IronOverlayManager = (function() {
+
+    var overlays = [];
+    var DEFAULT_Z = 10;
```

Comment:
this is the most annoying comment ever, but maybe `overlays` and `backdrops` should go together (i.e. one after the other), and `DEFAULT_Z` after them?